packages/flutter_tools/lib/src/preview_device.dart

```diff
@@ -29,7 +29,7 @@ BundleBuilder _defaultBundleBuilder() {
   return BundleBuilder();
 }
 
-class PreviewDeviceDiscovery extends DeviceDiscovery {
+class PreviewDeviceDiscovery extends PollingDeviceDiscovery {
```

Comment:
After this change, when I make the following request to `flutter daemon`:

```json
[{"id":"1","method":"device.enable","params":{}}]
```

I can see the following:

```json
[{"id":"1"}]
[{"event":"device.added","params":{"id":"preview","name":"preview","platform":"windows-x64","emulator":false,"category":"desktop","platformType":"custom","ephemeral":false,"emulatorId":null,"sdk":"preview","capabilities":{"hotReload":true,"hotRestart":true,"screenshot":false,"fastStart":false,"flutterExit":true,"hardwareRendering":true,"startPaused":true}}}]
[{"event":"device.added","params":{"id":"windows","name":"Windows","platform":"windows-x64","emulator":false,"category":"desktop","platformType":"windows","ephemeral":false,"emulatorId":null,"sdk":"Microsoft Windows [Version 10.0.22000.2538]","capabilities":{"hotReload":true,"hotRestart":true,"screenshot":false,"fastStart":false,"flutterExit":true,"hardwareRendering":true,"startPaused":true}}}]
[{"event":"device.added","params":{"id":"chrome","name":"Chrome","platform":"web-javascript","emulator":false,"category":"web","platformType":"web","ephemeral":false,"emulatorId":null,"sdk":"Google Chrome 120.0.6099.71","capabilities":{"hotReload":true,"hotRestart":true,"screenshot":false,"fastStart":false,"flutterExit":false,"hardwareRendering":true,"startPaused":true}}}]
[{"event":"device.added","params":{"id":"edge","name":"Edge","platform":"web-javascript","emulator":false,"category":"web","platformType":"web","ephemeral":false,"emulatorId":null,"sdk":"Microsoft Edge 120.0.2210.61","capabilities":{"hotReload":true,"hotRestart":true,"screenshot":false,"fastStart":false,"flutterExit":false,"hardwareRendering":true,"startPaused":true}}}]
```

Reply:
`platformType` here says `custom`, but in the response to `getSupportedPlatforms` it was `preview`?

(although again, I wonder whether it should have its own `platformType` as opposed to just being a new device using the existing platform type)

The name looks odd as "preview".. Name is the user-facing name, so I think it should at least have a capital `P` for consistency.

Reply:
I fixed it after making this comment, but good point about the capital P